setup.cfg

```diff
@@ -0,0 +1,21 @@
+[flake8]
+ignore = E266, W503
```

Comment:
Are these two temporary or do they mean to stay (i.e. we really don't care about them)?
It'd be good to add a comment if the ignore should be empty too.

Reply:
Sorry for not pointing this out but yes, we don't care about them and those ignores are meant to stay. We ignore those errors in other ETS projects as well.

Reply:
Thanks. I did not know that. Do you know if it is written somewhere? (it is not just some temporary thing ends up being thought of as permanent)

Reply:
:/ dont think its written anywhere afaik.

E266 - too many leading ‘#’ for block comment
W503 - line break before binary operator

https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes

E266 errors have been observed because of our use of comments of the style

```
#### object interface #########
```

Reply:
This is good to know. Thank you very much.